### PR TITLE
Launch blog post for `cargo-autoinherit`

### PR DESCRIPTION
**Pull request by LukeMathWalker**
A short-and-sweet announcement for the tool I wrote.

# Todos

- [x] Header image

### CODE REVIEW

**Review by marcoow — APPROVED** [review]
Nice 🙌 

Finally we get inheritance in Rust 🚎

File: src/posts/2024-03-18-cargo-autoinherit.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+When you have multiple packages in a Cargo workspace, you often end up depending
+on the same packages in multiple `Cargo.toml` files.  
+This duplication can become an issue:
+
+- When you want to update a dependency, you have to update it in multiple
+  places.
+- When you need to add a new dependency, you first have to check if it's already
+  used in another package of your workspace to keep versions in sync.
+
+This process it's error-prone and tedious.
```
Comment (BobrImperator) [inline src/posts/2024-03-18-cargo-autoinherit.md]: I think it should be:
```suggestion
This process is error-prone and tedious.  
```

File: src/posts/2024-03-18-cargo-autoinherit.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+title: "cargo-autoinherit: DRY up your workspace dependencies"
+authorHandle: algo_luca
+tags: [rust]
+bio: "Luca Palmieri, Principal Engineering Consultant"
+description:
+  "cargo-autoinherit is a new Cargo subcommand that converts your Cargo
+  workspace to use dependency inheritance, wherever possible. It helps you to
+  keep your dependencies DRY, reducing the risk of version mismatches and
+  unnecessary duplication."
+# TODO: Adjust header image
```
Comment (LukeMathWalker) [inline src/posts/2024-03-18-cargo-autoinherit.md]:
```suggestion
```

**Review by yaigrie — APPROVED** [review]

**Comment by marcoow** [comment]
@LukeMathWalker let's not merge with broken build 🙏

**Comment by LukeMathWalker** [comment]
Are you referring to Percy? @marcoow

**Comment by marcoow** [comment]
yes, Percy and Gravity in this case

**Comment by LukeMathWalker** [comment]
I'd love to get those greens, but I can't even see Percy reports 😅 I also don't understand why it goes red for a purely additive PR that doesn't modify an existing page?

**Pull request merged**